src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+0.5.162
```

Comment:
This has been used, please also update version in setup.py

Reply:
updated 0.5.162 in setup.py.